metashare/api/models.py

```diff
@@ -126,6 +127,13 @@ def instance_url(self):
         except (AttributeError, KeyError):
             return None
 
+    @property
+    def gh_username(self):
+        try:
+            return self.github_account.extra_data["login"]
+        except (AttributeError, KeyError):
+            return None
```

Comment:
No, there's not. I hadn't thought that `user.username` would reliably be the same, but it will be. However, I suppose there's the concern that someone changes their GitHub username, but in that case many things would get out of sync, and that's a larger issue.